Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -3385,42 +3474,24 @@ def get_machine_data(machine):
     return machine_data
 
 
-def get_file_statistics_command(client: MsClient, args: dict):
+def get_file_statistics_command(client: MsClient, args: dict) -> CommandResults:
     """Retrieves the statistics on the given file.
 
     Returns:
-        (str, dict, dict). Human readable, context, raw response
+        CommandResults.
     """
-    file_sha1 = args.get('file_hash')
-    response = client.get_file_statistics(file_sha1)
-    file_stat = get_file_statistics_context(response)
-    human_readable = tableToMarkdown(f'Statistics on {file_sha1} file:', file_stat, removeNull=True)
-    context_output = {
-        'Sha1': file_sha1,
-        'Statistics': file_stat
-    }
-    entry_context = {
-        'MicrosoftATP.FileStatistics(val.Sha1 === obj.Sha1)': context_output
-    }
-    return human_readable, entry_context, response
+    file_hash = args.get('file_hash', '')
+    response = client.get_file_statistics(file_hash)
+    file_stats_parser = FileStatisticsAPIParser.from_raw_response(response)
 
-
-def get_file_statistics_context(file_stat_response):
-    """Gets the file statistics response and returns it in context format.
-
-    Returns:
-        (dict). File statistics context
-    """
-    file_stat = assign_params(
-        OrgPrevalence=file_stat_response.get('orgPrevalence'),
-        OrgFirstSeen=file_stat_response.get('orgFirstSeen'),
-        OrgLastSeen=file_stat_response.get('orgLastSeen'),
-        GlobalPrevalence=file_stat_response.get('globalPrevalence'),
-        GlobalFirstObserved=file_stat_response.get('globalFirstObserved'),
-        GlobalLastObserved=file_stat_response.get('globalLastObserved'),
-        TopFileNames=file_stat_response.get('topFileNames'),
+    return CommandResults(
+        outputs_prefix='MicrosoftATP.FileStatistics',
+        outputs_key_field='Sha1',
+        indicator=file_stats_parser.to_file_indicator(file_hash),
+        readable_output=file_stats_parser.to_human_readable(file_hash),
+        outputs=file_stats_parser.to_context_output(),
```

Comment:
```suggestion
        indicator=file_stats.to_file_indicator(file_hash),
        readable_output=file_stats.to_human_readable(file_hash),
        outputs=file_stats.to_context_output(),
```

Reply:
Fixed in commit d19600a189c37cd368f75dcb76ca7b2f84b28887